libs/user-identity-service/src/test/java/uk/gov/di/ipv/core/library/service/UserIdentityServiceTest.java

```diff
@@ -204,7 +209,7 @@ void areVCsCorrelatedReturnFalseWhenNamesDiffer() throws Exception {
                 List.of(
                         TestFixtures.createVerifiableCredential(
                                 USER_ID_1,
-                                ADDRESS.getId(),
+                                PASSPORT.getId(),
```

Comment:
Why make the VCs all from the same issuer? I don't think it actually matters for these tests, but having different issuers might be more realistic and potentially catch some weird edge case?